platform/pulpcore/app/viewsets/repository.py

```diff
@@ -51,6 +51,19 @@ def importers(self, request, name):
         serializer = ImporterSerializer(page, many=True, context={'request': request})
         return paginator.get_paginated_response(serializer.data)
 
+
+    def update(self, request, partial=True):
```

Comment:
Actually now that I think about it PUT/PATCH RFCs already note that PUT should include all parameter by design [0]. Since this is a part of the protocol design, I don't think we need extra documentation on it.

[0] https://stackoverflow.com/questions/28459418/rest-api-put-vs-patch-with-real-life-examples